main.go

```diff
@@ -141,19 +160,19 @@ func uploadFileListWithRetry(assets []releaseAsset, client *github.Client, owner
 			return fmt.Errorf("failed to open file (%s), error: %s", asset.path, err)
 		}
 
-		if err := uploadFileWithRetry(asset.path, asset.displayFileName, fi, client, owner, repo, id); err != nil {
+		if err := uploadFileWithRetry(GetUploader(uploadAsset, 3, 5000), asset.path, asset.displayFileName, fi, client, owner, repo, id); err != nil {
 			return err
 		}
 	}
 	return nil
 }
 
-func uploadFileWithRetry(filePath string, fileName string, fi *os.File, client *github.Client, owner string, repo string, id int64) error {
-	return retry.Times(3).Wait(5 * time.Second).Try(func(attempt uint) error {
+func uploadFileWithRetry(uploader *Uploader, filePath string, fileName string, fi *os.File, client *github.Client, owner string, repo string, id int64) error {
```

Comment:
Could add the number of retries as a return value, as matching on the log contents in the tests could break later.